lib/dry/schema/key.rb

```diff
@@ -46,6 +46,8 @@ def read(source)
           yield(source[name])
         elsif source.key?(coerced_name)
           yield(source[coerced_name])
+        elsif source.key?(name.to_s)
+          yield(source[name.to_s])
```

Comment:
Why is this needed?

Reply:
This was a big fail, because I was testing a non-params schema by an accident, so all of my tests were failing when I passed any string key :facepalm:  

I'm currently refactoring it.